src/const.py

```diff
@@ -25,6 +25,7 @@
 # e.g RGW_CONF_TMPL will be /opt/seagate/cortx/rgw/cof/cortx_rgw.conf
 RGW_CONF_FILE = f'cortx_{COMPONENT_NAME}.conf'
 # e.g. RGW_CONFI_FILE path will be cortx_rgw.conf
+RGW_CRASHDUMP_DIR = '/var/log/ceph/crash'
```

Comment:
Isnt there any core dump specific pvc storage which we use. @sachitanands where s3server used to collect its cores, same we should be reusing

Reply:
we need to find way to configure crash dump file path in rgw config file.
As of now as per ceph documentation, this is the path where crash files are getting dumped.

Reply:
@gargpalak048  could you pls try " ceph crash info --no-mon-config" command once.
we need to explore on this but agree its not straight forward since crash is centralized for ceph.
for this change I would suggest go ahead and merge.
cc @cdeshmukh

Reply:
@sachitanands " ceph crash info --no-mon-config" command gives the same output.

Reply:
yeah its fine, though make sure support bundle picks up core files from this location.
If service crashes, container may terminate, making that crash dir inaccessible too, so we may need to have PVC for this too, like we need for log retention. Make note and address this while we address PVC requirements for RGW logs
@sachitanands @selvakumaar5496